documentation/sphinx/source/bulkdump.rst

```diff
@@ -69,7 +69,7 @@ ManagementAPI provides following interfaces to do the operations:
 1. Submit a job: submitBulkDumpJob(BulkDumpState job); // For generating the input job metadata, see the point 4.
 2. Clear a job: clearBulkDumpJob();
 3. Enable the feature: setBulkDumpMode(int mode); // Set mode = 1 to enable; Set mode = 0 to disable.
-4. BulkDump job metadata is generated by newBulkDumpTaskLocalSST(KeyRange range, std::string remoteRoot); // Will include more APIs to generate the metadata as the funcationality expands (sp of functionality).
+4. BulkDump job metadata is generated by newBulkDumpJobLocalSST(KeyRange range, std::string remoteRoot); // Will include more APIs to generate the metadata as the funcationality expands (sp of functionality).
```

Comment:
Do we need the 'Local' and/or 'SST'? How it is persisted is of no concern to the caller? And when it is not-local, how will the metadata be different? It'll be the same no?  So are 'Local' and 'SST' unnecessary (and perhaps obstacles as we go forward)